helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -50,6 +50,7 @@ spec:
       labels:
         {{- include "hpcc.addStandardLabels" (dict "root" $ "component" "dafilesrv" "name" "dafilesrv" "instance" .name) | indent 8 }}
         server: {{ .name | quote }}
+        helmVersion: 8.10.3-closedown0
```

Comment:
helmVersion was missing in dafilesrv.yaml - this matches the same setting in other template yamls.
It/they are manipulated on a rc and gold release.